src/main/scala/dpla/ingestion3/mappers/providers/LcMapping.scala

```diff
@@ -0,0 +1,165 @@
+package dpla.ingestion3.mappers.providers
+
+import java.net.URI
+
+import dpla.ingestion3.enrichments.normalizations.StringNormalizationUtils._
+import dpla.ingestion3.mappers.utils.{Document, IdMinter, JsonExtractor, Mapping}
+import dpla.ingestion3.model.DplaMapData._
+import dpla.ingestion3.model.{EdmAgent, _}
+import dpla.ingestion3.utils.Utils
+import org.json4s.JValue
+import org.json4s.JsonDSL._
+import org.json4s.jackson.JsonMethods._
+
+
+// TODO Should there be an implicit unwrapping of JSON values when calling extract*(JValue)
+
+class LcMapping() extends Mapping[JValue] with IdMinter[JValue] with JsonExtractor {
+
+  // ID minting functions
+  override def useProviderName: Boolean = true
+
+  // Hard coded to prevent accidental changes to base ID
+  override def getProviderName: String =
+    "loc"
+
+  override def getProviderId(implicit data: Document[JValue]): String =
+    extractString(data.get \ "item" \ "id") // TODO confirm basis field for DPLA ID
```

Comment:
@guegueng The ID minted in heidrun came from `library_of_congress_control_number` field whereas i3 is using `id`. The values overlap but I want to confirm that the control number is preferred for the sake of continuity between heidrun and i3. 

`library_of_congress_control_number = 73691632`
`id = http://www.loc.gov/item/73691632/`

https://github.com/dpla/heidrun/commit/6ab3f38f62156682a04579af8a98191bd072a05f#diff-1a4317d11c81f5bff6982f05e748c4bfR105